front/app/api/invites/useDeleteInvite.ts

```diff
@@ -0,0 +1,24 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import fetcher from 'utils/cl-react-query/fetcher';
+import invitesKeys from './keys';
+
+const deleteInvite = ({ inviteId }: { inviteId: string }) =>
```

Comment:
```suggestion
const deleteInvite = (inviteId: string)=>
```
Do we have a reason to not have this simpler version? Like we do for e.g. `useDeleteArea`.

Reply:
No, this should be simplified 👍